data/ecosystem/distributions.yaml

```
@@ -83,3 +83,19 @@
   url: https://github.com/signalfx/splunk-otel-go
   docsUrl: https://docs.splunk.com/observability/en/gdi/get-data-in/application/go/get-started.html
   components: [Go]
+- name: Azure Monitor Distribution of OpenTelemetry ASP.NET Core
+  url: https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore
+  docsUrl: https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable
```

```suggestion
  docsUrl: https://learn.microsoft.com/azure/azure-monitor/app/opentelemetry-enable?tabs=aspnetcore
```

Could we put more specific links?

---

Updated.